src-graphql/images-query.js

```diff
@@ -0,0 +1,9 @@
+export default function imagesQuery(fields = ['id', 'src', 'altText']) {
```

Comment:
consistency in file naming and method naming. Some of these are plural and some singular.